internal/dataplane/fallback/graph_dependencies_kong.go

```diff
@@ -9,13 +11,33 @@ import (
 	incubatorv1alpha1 "github.com/kong/kubernetes-ingress-controller/v3/pkg/apis/incubator/v1alpha1"
 )
 
-// TODO: https://github.com/Kong/kubernetes-ingress-controller/issues/5929
-func resolveKongPluginDependencies(_ store.CacheStores, _ *kongv1.KongPlugin) []client.Object {
+// resolveKongPluginDependencies resolves potential dependencies for a KongPlugin object:
+// - Secret.
+func resolveKongPluginDependencies(cache store.CacheStores, kongPlugin *kongv1.KongPlugin) []client.Object {
+	cf := kongPlugin.ConfigFrom
+	if cf == nil {
+		return nil
+	}
+	if secret, exists, err := cache.Secret.GetByKey(
+		fmt.Sprintf("%s/%s", kongPlugin.Namespace, cf.SecretValue.Secret),
+	); err == nil && exists {
+		return []client.Object{secret.(client.Object)}
+	}
 	return nil
 }
 
-// TODO: https://github.com/Kong/kubernetes-ingress-controller/issues/5929
-func resolveKongClusterPluginDependencies(_ store.CacheStores, _ *kongv1.KongClusterPlugin) []client.Object {
+// resolveKongClusterPluginDependencies resolves potential dependencies for a KongClusterPlugin object:
+// - Secret.
+func resolveKongClusterPluginDependencies(cache store.CacheStores, kongClusterPlugin *kongv1.KongClusterPlugin) []client.Object {
+	cf := kongClusterPlugin.ConfigFrom
```

Comment:
Ditto - `ConfigPatches` references.

Reply:
Thanks for catching, I totally forgot about this